### ISSUE TITLE
Use of uninitialized value $changedframes{"TRACKNUMBER"}

### ISSUE THREAD

**Issue by BoxSoft** [robinbowes/flac2mp3#46]
I'm getting the following error:

Use of uninitialized value $changedframes{"TRACKNUMBER"} in multiplication (*) at C:\Program Files (x86)\flac2mp3\flac2mp3.pl line 226.

Any suggestions?

**Comment by robinbowes** [comment]
What version are you using? Does an mp3 file exist already? Are you able to upload the flac file and the mp3 (if any)?

**Comment by BoxSoft** [comment]
I downloaded the latest today, although I've been experiencing this same error for a long time.  (That was why I downloaded today.)  Unfortunately, it's difficult to pin down exactly when it's happening, as it's processing my full music collection (over 20,000 songs).  It happens between two folders, so I'm not sure which of the hundreds folders is causing it to fail.

BTW, it also continually reprocesses a small subset of tracks each time.  Sometimes it's all FLACs within a folder, and sometimes only a few.  I'm not sure why it does that.  Does your tool store some static information somewhere, or does it derive everything it needs purely from the directory scan?

Would it be worthwhile nuking my MP3 folder and rerunning it from scratch?

**Comment by robinbowes** [comment]
Did the latest upgrade fix your original problem?

R.

**Comment by BoxSoft** [comment]
No, all errors experienced before the update appear to be the same after.  BTW, I decided to try nuking the entire MP3 folder and reprocessing everything, just in case the newer version would have better luck with a clean slate.  Is there any additional information that's stored elsewhere, or is everything derived by comparing the two folders?
